components/PrivacyPage/PrivacyPage.tsx

```diff
@@ -0,0 +1,1840 @@
+            <div>
+              iii. Commercial Use. Subject to your continued compliance with these Terms, Defi
+              Entertainment Inc.
+              <br />
+              grants you an unlimited, worldwide license to use, copy, and display the purchased Art
+              for the purpose
+              <br />
+              of creating derivative works based upon the Art (“Commercial Use”). Examples of such
+              Commercial
+              <br />
+              Use would e.g. be the use of the Art to produce and sell merchandise products
+              (T-Shirts etc.)
+              <br />
+              displaying copies of the Art. For the sake of clarity, nothing in this Section will be
+              deemed to restrict
+              <br />
+              you from (i) owning or operating a marketplace that permits the use and sale of
+              Cre8ors generally,
+              <br />
+              provided that the marketplace cryptographically verifies each Cre8or owner’s rights to
+              display the Art
+              <br />
+              for their Cre8or to ensure that only the actual owner can display the Art; (ii) owning
+              or operating a<br />
+              third party website or application that permits the inclusion, involvement, or
+              participation of Cre8ors
+              <br />
+              generally, provided that the third party website or application cryptographically
+              verifies each Cre8or
+              <br />
+              owner’s rights to display the Art for their Cre8or to ensure that only the actual
+              owner can display the
+              <br />
+              Art, and provided that the Art is no longer visible once the owner of the Purchased
+              Cre8or leaves the
+              <br />
+              website/application; or (iii) earning revenue from any of the foregoing.
+            </div>
+          </div>
+        </>
+      ),
+    },
+  ]
+
+  const containerRef = useRef<any>()
+
+  const isMobile = useMediaQuery("(max-width: 799px)")
+
+  const { scrollY } = useScroll({ container: containerRef })
+
+  const { isScrollEnded } = useScrollEnded({
+    ref: containerRef,
+    scrollY,
+  })
+
+  return (
+    <Layout type="contained">
+      <div className="relative pb-[2rem] h-[100vh] w-[100vw] overflow-y-scroll" ref={containerRef}>
+        {!isScrollEnded && (
+          <div
+            className="fixed 
+              w-[100vw] h-[265px] 
+              left-0 bottom-0
+              pointer-events-none
+              bg-gradient-to-t from-[white] dark:from-[black] to-[transparent]
+              z-[20]"
+          />
+        )}
+        <div className="relative w-full">
+          <div
+            className="font-eigerdals 
+              text-[36px] md:text-[65px] 
+              pt-[10rem] pb-[40px] 
+              dark:text-[white] 
+              mx-4 md:mx-12"
+          >
+            Terms of Service
```

Comment:
this should say `Privacy Policy` right?
```suggestion
            Privacy Policy
```